ts/call-action-form.tsx

```diff
@@ -104,9 +104,11 @@ export class CallActionForm extends React.Component<Props, State> {
 			} as State);
 			this.props.setModal("loading");
 			var campaign = this.getCampaign(this.props.campaignId);
+			var zipField = (document.getElementsByName('input_zip').item(0) as HTMLInputElement);
```

Comment:
so the way I would handle this with the current architecture is to add a new zip parameter to the setModal function, and in PetitionForm do this.props.setModal("call", this.state.input_zip); this call then would set a zip state variable in BFTNFormFlow which then gets passed to CallActionForm. make sense?